### PR TITLE
[MIG][14.0] stock_picking_show_return: Migration to v14.0

### PR DESCRIPTION
**Pull request by hugho-ad**

### CODE REVIEW

**Comment by hugho-ad** [comment]
FYI @Tardo @carlosdauden @rousseldenis

File: stock_picking_show_return/i18n/es.po [hunk 1]
```diff
@@ -1,45 +1,56 @@
...
+#. module: stock_picking_show_return
+#: model:ir.model.fields,field_description:stock_picking_show_return.field_stock_picking____last_update
+msgid "Last Modified on"
+msgstr ""
 
 #. module: stock_picking_show_return
 #: model:ir.model.fields,field_description:stock_picking_show_return.field_stock_picking__returned_ids
 msgid "Returned pickings"
-msgstr "Albaranes devueltos"
+msgstr ""
```
Comment (sebalix) [inline stock_picking_show_return/i18n/es.po]: Is it intended to drop these translations during the migration?

Comment (hugho-ad) [reply stock_picking_show_return/i18n/es.po]: No, it's not, solved

**Comment by hugho-ad** [comment]
@moylop260

could you review pls?

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by moylop260 — APPROVED** [review]
👍

**Pull request merged**